Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    bbbevents (1.1.0)
```

Comment:
As this is not a big change, I would tag it as 1.2.0 or even 1.1.1. I don't think there is need for a major release

Reply:
Changed version to 1.2.0

Reply:
This is likely break compatibility with our statistics html page generation stuff. If it's breaking compatibility, it should be a major version change.